include/git2/status.h

```diff
@@ -173,12 +173,17 @@ typedef enum {
  * The `pathspec` is an array of path patterns to match (using
```

Comment:
Pleas mind the preferred commit message style. First again, the area-prefix ("status" in this case), and then please also mind that we wrap the body at 80 characters. Also, please try to make the message more imperative instead arguing in place of the project, not yourself.

Reply:
I've now created PR #4422 to document our style. There's still two minor violations:

1. the first letter after the area-prefix is usually lowercased
2. the subject is some characters too long

Proposed: "status: add option to compare to trees other than head". You could add some reasoning in the commit message body, but that's only optional.